packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,96 @@
+participant "Resource Server (Your App)" as app
+
+user -> client: Start auth with user info
+client -> client: Generate PKCE code verifier & challenge
+client -> okta: Authorization request w/ code_challenge, scopes, and user info
+okta -> okta: Remediation required
+okta -> client: Sends interaction_handle in response (for required interaction)
+user <-> client: Remedial interaction
+client <-> oie: Remedial interaction w/ interaction_handle
+note right: Possible multiple remedial steps required
+user <-> client: Remedial interaction
+client -> oie: Remedial interaction w/ interaction_handle
+oie -> oie: Remedial steps completed
+oie -> client: Send interaction_code in response
+client -> okta: Send interaction_code, client ID, code_verifier to /token
+okta -> okta: Evaluates PKCE code
+okta -> client: Access token (and optionally refresh token)
+client -> app: Request with access token
+app -> client: Response
+@enduml
+
+ -->
+
+![Interaction Code flow sequence diagram](/img/authorization/interaction-code-grant-flow.png)
+
+* The Interaction Code flow can start with minimal user information. For example, the user (resource owner) may only provide the client app with their username. Alternatively, the client can also begin the flow without any user information for a passwordless or a social sign-in experience.
+
+* The client app generates the PKCE code verifier & code challenge.
+
+* The client app begins interaction with the Authorization Server, providing any context it may have, such as a login hint, as well as sending the code challenge in a request for authorization of certain scopes to the Okta Authorization Server.
+
+  > **Note:** A confidential client authenticates with the Authorization Server while a public client (like the Sign-In Widget) identifies itself to the Authorization Server. Both must provide the PKCE code challenge.
+
+* The Authorization Server sends the `interaction_handle` parameter in the body of a response to the client app.
+
+  > **Note:** The `interaction_handle` is used to continue the interaction directly with the Identity Engine. This is why the client, either confidential or public, needs to be registered with the Identity Engine to perform this direct interaction.
+
+* The client sends the `interaction_handle` to the Identity Engine and in return, the Identity Engine sends any required remediation steps to the client. The client begins an interactive flow with the Identity Engine and the resource owner, handling any type of interaction required by the Identity Engine (the remedial information is provided by the user to the client).
+
+* When the remediation steps are completed, the Identity Engine sends back a success response and includes the `interaction_code` to the client.
+
+  > **Note:** The Interaction Code has a maximum lifetime of 60 seconds.
+
+* The client sends the code verifier and the `interaction_code` to the Authorization Server (`/${authServerId}/token` endpoint) to exchange for tokens.
```

Comment:
The url provided is related to a Custom Authorization Server, but interaction code flow is supported by both Org and Custom Authorization Server. I'm not sure if that need to be specified here or in the `Overview` section.

Reply:
I will add Org and Custom AS support in the Overview section. Also, I'll remove the endpoint reference here.